src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/ObTableObjType.java

```diff
@@ -141,19 +141,58 @@ public int getEncodedSize(ObObj obj) {
     // 15 ObTableUInt32Type
     // 16 ObTableUInt64Type
 
-    ObTableInvalidType(17) {
+    ObTableTinyTextType(17) {
+        public void decode(ByteBuf buf, ObObj obj) {
+            decodeWithUtf8(buf, obj);
+        }
+    },
+
+    ObTableTextType(18) {
+        public void decode(ByteBuf buf, ObObj obj) {
+            decodeWithUtf8(buf, obj);
+        }
+    },
+
+    ObTableMediumTextType(19) {
+        public void decode(ByteBuf buf, ObObj obj) {
+            decodeWithUtf8(buf, obj);
+        }
+    },
+
+    ObTableLongTextType(20) {
+        public void decode(ByteBuf buf, ObObj obj) {
+            decodeWithUtf8(buf, obj);
+        }
+    },
+
+    ObTableTinyBlobType(21) {
```

Comment:
Why 21-24 don't need override `decode()`?

Reply:
just inherit decode function from the ObTableObjType enum class is enough, it will:
1. decode 1 byte table object type
2. get the ob obj type according the mapping from table object type to ob obj type
3. get the default meta of the ob obj type and decode value using ob obj type's decode method